src/controllers/attachment/index.ts

```diff
@@ -91,28 +124,27 @@ export class attachment extends Controller {
   @Get('/{id}')
   @Response<NotFound>(404)
   @Response<BadRequest>(400)
+  @Produces('application/octet-stream')
+  @Produces('application/json')
   @SuccessResponse(200)
-  public async getById(
-    @Request() req: express.Request,
-    @Path() id: string,
-    @Header('return-type') type: 'json' | 'file'
-  ): Promise<unknown | Readable> {
+  public async getById(@Request() req: express.Request, @Path() id: string): Promise<unknown | Readable> {
     this.log.debug(`attempting to retrieve ${id} attachment`)
-    const { accept } = req.headers
     const [attachment] = await this.db.attachment().where({ id })
     if (!attachment) throw new NotFound('attachment')
     const { filename, binary_blob } = attachment
 
-    // log and default to octect-stream
-    if (type === 'json' || accept === 'application/json') {
-      this.setHeader('content-type', 'application/json')
-      try {
-        return JSON.parse(binary_blob)
-      } catch (err) {
-        this.log.warn('requested type failed, returning as octet')
+    const orderedAccept = parseAccept(req.headers.accept || '*/*')
```

Comment:
under what instance we would get accept headers containing multiple. Is this is required by other parts of DSCP?

Reply:
it's part of the Accept header standard https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept